notebooks/official/feature_store/online_feature_serving_bigquery_data_with_feature_store_2_0.ipynb

```diff
@@ -0,0 +1,1167 @@
+{
```

Comment:
nit: the title probably can be something like feature store 2.0 tutorial (vector retrieval) instead of (goku), because goku is an internal code name.

Reply:
Understood. We didn't use goku in the notebook. This title of the notebook has been changed to "Online feature serving and vector retrieval BigQuery data with Vertex AI Feature Store 2.0". Please let me know if it'd work. Thanks.